File: PAT/初级/1001.py
def getAns(n):
    cnt = 0
    while n > 1:
        if n % 2 == 1:
            n = (3 * n + 1) / 2
        else:
            n /= 2
        cnt += 1
    return cnt


n = int(input())
print(getAns(n))

File: PAT/初级/1001.cpp
#include <cstdio>

int getAns(int n) {
    int cnt = 0;
    while (n > 1) {
        ++cnt;
        if (n % 2) {
            n = (n * 3 + 1) / 2;
        } else {
            n /= 2;
        }
    }
    return cnt;
}

int main() {
    int n;
    scanf("%d", &n);
    printf("%d\n", getAns(n));
    return 0;
}